build/agent/nginx.conf

```diff
@@ -0,0 +1,29 @@
+load_module /usr/lib/nginx/modules/ngx_http_js_module.so;
+
+events {}
+
+pid /etc/nginx/nginx.pid;
+
+error_log /var/log/nginx/error.log debug;
```

Comment:
do we need to use file path in the error_log and access_log directives? why not `stderr` and `stdout` ?

Reply:
This is required by the agent so that it can parse the logs for metrics. The logs are still output to `stdout`.

Reply:
hm. I wonder it works - is the agent still capable of reading those files event the logs are still output to stdout?

Reply:
Yeah, I enabled trace logs and verified that the agent is collecting metrics from the access logs. 

I think it works because the nginx Dockerfile forwards access and error logs to the docker log collector: https://github.com/nginxinc/docker-nginx/blob/761fffeba0d867d6e80d38998073e0eaa456bb02/stable/alpine-slim/Dockerfile#L105.